extensions/amp-image-lightbox/0.1/amp-image-lightbox.js

```diff
@@ -527,14 +527,18 @@ export class ImageViewer {
   onZoom_(scale, deltaX, deltaY, animate) {
     const newScale = this.boundScale_(scale, true);
     if (newScale == this.scale_) {
-      return;
+      return Promise.resolve();
```

Comment:
There are two modes: animated and non-animated. Only animated mode returns promise. The callers know this because they pass `animate: true` argument. Not exactly the nicest design here. But given the private methods I figured this was better than adding extra methods to disambiguate them.